tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import ssl
+
+import pytest
+import trustme
```

Comment:
this should be added to test dependencies

Reply:
oops - added!